src/azure-cli/azure/cli/command_modules/containerapp/_utils.py

```diff
@@ -509,12 +509,21 @@ def _update_revision_env_secretrefs(containers, name):
 
 
 def store_as_secret_and_return_secret_ref(secrets_list, registry_user, registry_server, registry_pass, update_existing_secret=False, disable_warnings=False):
+    def make_dns1123_compliant(name):
+        logger.debug(f"Updating username '{name}' to DNS1123-compliant format.")
```

Comment:
Hi @william-msft 
`make_dns1123_compliant` is only for construct registry_secret_name, I think this debug message is a little bit confuse.

```suggestion
        logger.debug(f"To construct the registry secret name, format the username '{name}' to DNS1123-compliant format.")
```